app/src/main/java/com/khinthirisoe/bakingapp/ui/steps/StepsActivity.kt

```diff
@@ -7,66 +7,50 @@ import android.os.Bundle
 import androidx.appcompat.app.AppCompatActivity
 import com.khinthirisoe.bakingapp.R
 import com.khinthirisoe.bakingapp.data.model.Step
+import com.khinthirisoe.bakingapp.ui.steps.player.MediaPlayerImpl
 import kotlinx.android.synthetic.main.activity_steps.*
 
-class StepsActivity : AppCompatActivity(), StepsContract.View {
+class StepsActivity : AppCompatActivity() {
+
+    private lateinit var mediaPlayerImpl: MediaPlayerImpl
```

Comment:
@KhinThiriSoe MediaPlayerImpl is also not required. It's easier without it.

Reply:
fixed it.